src/script/media/CanvasMediaStreamMixer.ts

```diff
@@ -0,0 +1,317 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+// Canvas configuration
```

Comment:
This you can not know, but at AVS, we put a lot of effort into transmitting screen shares in the best possible quality. Here You’re drawing a canvas with fixed dimensions. That means if someone captures a screen with 4000 x 2000 dimensions, you’re scaling it down. This results in a loss of resolution.

I know that on the receiver side, the screen share is also scaled down to fit the dimensions (one of the reasons why we don’t have 100% best screen share quality). Nevertheless, we should avoid downscaling on the sender side.